pkg/kv/kvserver/liveness/liveness.go

```diff
@@ -676,7 +676,8 @@ func (nl *NodeLiveness) Start(ctx context.Context, opts NodeLivenessStartOptions
 		defer sp.Finish()
 
 		incrementEpoch := true
-		ticker := time.NewTicker(nl.heartbeatInterval)
+		interval := nl.livenessThreshold - nl.renewalDuration
```

Comment:
[optional] I think renaming this as `heartbeatInterval` is slightly clearer.

Reply:
Done.